packages/slice-machine/components/CodeBlock/index.tsx

```diff
@@ -2,8 +2,12 @@ import React from "react";
 import { type ThemeUIStyleObject, Flex, useThemeUI } from "theme-ui";
 import hljs from "highlight.js";
 
+import { svelte } from "@lib/hljs/svelte";
+
 const DEFAULT_LANGUAGES = ["javascript", "bash", "xml", "html", "json"];
 
+hljs.registerLanguage("svelte", svelte);
```

Comment:
Interesting. I didn't realize we had two separate code highlighters, one using `highlight.js` and the other using `prism`. Yes, I will add Svelte language support to `packages/slice-machine/src/components/CodeBlock/CodeBlock.tsx`.

In the future, should we choose one syntax highlighter, and maybe use a more modern one like [Shiki](https://github.com/shikijs/shiki)?

Reply:
Adding Svelte support to `packages/slice-machine/src/components/CodeBlock/CodeBlock.tsx` is tricker than I thought since it doesn't use standard Prism language registration.

I will leave this comment open and try to get back to it before release. If it cannot be done immediately, page snippet code won't have syntax highlighting, but field snippets will.